app/views/curation_concerns/base/_attribute_rows.html.erb

```diff
@@ -7,6 +7,7 @@
 <%= presenter.attribute_to_html(:identifier, render_as: :linked, search_field: 'identifier_tesim') %>
 <%= presenter.attribute_to_html(:keyword, render_as: :faceted) %>
 <%= presenter.attribute_to_html(:date_created, render_as: :linked, search_field: 'date_created_tesim') %>
-<%= presenter.attribute_to_html(:based_near, render_as: :faceted) %>
+<%= presenter.attribute_to_html(:based_near, label: 'Location', render_as: :faceted) %>
```

Comment:
If you do this, it doesn't use i18n, which prevents translations to other languages.